go-controller/pkg/ovn/master.go

```diff
@@ -723,7 +723,7 @@ func (oc *Controller) clearInitialNodeNetworkUnavailableCondition(origNode, newN
 	resultErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
 		var err error
 
-		oldNode, err := oc.kube.GetNode(origNode.Name)
+		oldNode, err := oc.watchFactory.GetNode(origNode.Name)
```

Comment:
@abhat this one we want to keep hitting up the Kube API directly, since here we're trying to make sure we have the latest version of the object before we update it.

Reply:
Ok will switch back to using the `kube interface`